libethereum/Executive.cpp

```diff
@@ -391,6 +400,8 @@ bool Executive::go(OnOpFunc const& _onOp)
 			auto vm = _onOp ? VMFactory::create(VMKind::Interpreter) : VMFactory::create();
 			if (m_isCreation)
 			{
+				if (m_s.addressHasCode(m_newAddress))
```

Comment:
That stuff will basically never return true before Metropolis. but yeah that worth testing only after metro